### PR TITLE
First attempt at norm=permillion

### PR DESCRIPTION
**Pull request by dmaymudes**
Please don't accidentally accept this pull request--it doesn't actually work, because the region label in the series passed to apply_norm isn't in the same format as the population labels I have.

Let me know what you suggest; I could figure out how to change the population.js file to have labels like "Switzerland" and "MA" rather than "SWI" and "MA, US", or I could figure out a way to plumb the more canonical labels through.

I also see that timeseries-byLocation.json has "Barnstable County, MA, US" where whatever datasource the current version of the chart uses has "Barnstable, MA" so maybe further cleanup would be required.

I guess nothing so terrible would happen if you *did* accept it, because it doesn't do anything yet without norm=permillion.

### CODE REVIEW

**Comment by davidbau** [comment]
Very cool!  So the main issue is that we need to normalize location names between JHU CSSE and CDS data sources.  This is something we should do for other reasons, too, because we can get much higher-quality data if we merge the historical CSSE and CDS time series data.

Captured the things that need to be done in issue #34.

**Comment by dmaymudes** [comment]
I think this change works now, let me know if you have further questions.

The JSON file with population entries got much bigger, now the data feed has 9000 entries.

**Comment by davidbau** [comment]
Sorry for the slow reply.  A few requests
- Let's called it "norm=pop" instead of "permillion".  I.e., normalized by population.  Per-million does seem like the right units.
- County-level populations - visit this page - http://localhost:8000/#/?domain=NY&norm=permillion&advanced=1.  I think the file contains all the needed county-level population numbers, but for some reason they're not getting lined up.
- Omit lines when we don't have population data instead of drawing the wrong line.  If for whatever reason something doesn't line up, it would be better to filter out the data series so it's just not shown, instead of showing something with the wrong units.

File: lib/src/make_pop.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+pop = {}
+
+present = 0
+absent = 0
+
+for k,v in data.items():
+    if k in country_map:
+        k = country_map[k]
+    k = k.replace(', USA', '')
+    k = k.replace(' County, ', ', ')
```
Comment (davidbau) [inline lib/src/make_pop.py:48]: Might want to also check " Parish" and " Borough".  In merging CDS data earlier this week I have a bunch of other messy data cleaning cases.

Comment (davidbau) [reply lib/src/make_pop.py:48]: A catalog of CDS spelling differences (and errors) is encoded in this messy cleaning code:
https://github.com/davidbau/covid-19-chart/blob/merged_feeds/lib/load_flat_cds.js

Comment (dmaymudes) [reply lib/src/make_pop.py:48]: OK, this resolves "Parish" and "Borough", so maybe it's usable as a first attempt.

really what we should do is intersect the labels from the two data sources and print a list of "entries without populations" and "populations without matching entries" which would make it much easier to clean up remaining differences.

(or have a single name canonicalization library, but that's annoying to do across languages. I could change the make_pop code to be JS, I guess.

**Pull request merged**